Header  header       # timestamp

float64 battery      # the current battery level, 0-100

string  power        # ON/OFF state
string  device_mode  # pressMode, switchMode, or customizeMode
